cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/objectstorage/ObjectStorageValidateRequest.java

```diff
@@ -175,6 +190,13 @@ public Builder withMockSettings(String region, String adminGroupName) {
             return this;
         }
 
+        public Builder withAzureParameters(String singleResourceGroupName) {
```

Comment:
It does not make a difference now, but I suggest changing the signature to take an argument of type `AzureParameters` instead.

Reply:
I would leave this one as long as the number of parameters are limited because this way we could create builder method for the important scenarios limiting the chance of not setting or setting too many parameters and this way we can keep AzureParameters as implementation detail.